spraycharles.py

```diff
@@ -304,8 +323,16 @@ def main(passlist, userlist, host, module, path, csvfile, attempts, interval, eq
         login_attempts += 1
     
     # analyze the results to point out possible hits
-    analyzer = analyze.Analyzer(csvfile)
-    analyzer.analyze()
+    analyzer = analyze.Analyzer(csvfile, notify, webhook, host)
+    success = analyzer.analyze()
+
+    # Pausing if specified by user before continuing with spray
+    if success is True and pause:
+        print()
+        colors.color_print('[+] Successful login potentially identified. Pausing...', colors.yellow)
+        print()
+        input('Press enter to continue:')
```

Comment:
I believe this is out of the `for` loop and will trigger a pause on a hit even when all that's left to do is exit the script. We can probably delete this block, if you have the same assessment.